### ISSUE TITLE
Kernel constants not being set for FaultCohesiveKin

### ISSUE THREAD

**Issue by baagaard-usgs** [geodynamics/pylith#120]
`FaultCoheisveKin` has `_updateKernelConstants` but this does not appear to be called.

# Test case

```
cd examples/3d/subduction
pylith step02_coseismic.cfg MORE_CONFIG_FILES?
```

**Comment by baagaard-usgs** [comment]
@knepley I can see `pylith::feassemble::Integrator::_setKernelConstants()` being called. I get `constants == NULL` in the kernel. I think the issue is that this is the fault and we are setting the constants in the wrong DS. How do we update [line 141](https://github.com/geodynamics/pylith/blob/baagaard/reorganize-examples/libsrc/pylith/feassemble/Integrator.cc#L141) of `Integrator.cc` to handle the fault integration case?

**Comment by baagaard-usgs** [comment]
`DMGetNumDS()` (Number of DS in the problem)
Loop over each DS  and call `DMGetRegionNumDS`

**Comment by baagaard-usgs** [comment]
Fixed in bb04b107ad1bdb96e1e463e91ee91b8791c9c04b.
